src/types/index.ts

```diff
@@ -9,6 +9,13 @@ export interface TypeMapping {
     fundraiserDetails: EveryFundraiserRaisedDetails;
     fundraiserCreationResponse: FundraiserCreationResponse;
     everyExistingFundraiserInfo: EveryExistingFundraiserInfo;
+    fundraiserStatus: FundraiserStatus;
+}
+
+export enum FundraiserStatus {
+  Active = 'active',
+  Completed = 'completed',
+  Expired = 'expired'
```

Comment:
What's the functional difference between these two? Worth documenting the definitions here.